examples/joy-ui-nextjs/src/components/ThemeRegistry/theme.ts

```diff
@@ -0,0 +1,45 @@
+import { extendTheme } from '@mui/joy/styles';
+import { Inter, Source_Code_Pro } from 'next/font/google';
+
+const inter = Inter({
+  subsets: ['latin'],
```

Comment:
```suggestion
  subsets: ['latin'],
  weight: ['300', '400', '500', '600', '700'],
```
Does it need to specify weight?

Reply:
Inter seems to be a variable font, in which the `weight` can be omitted.

Reply:
Thanks for the info!